jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+def run(params) {
+    timestamps {
+        // Start pipeline with default values
+        arch= 'x86_64'
+        built = false
+        deployed = false
+        tests_passed = false
+        sumaform_backend = 'libvirt'
+        terraform_bin = '/usr/bin/terraform'
+        terraform_bin_plugins = '/usr/bin'
+        service_pack_migration = false
+//        terracumber_gitrepo = 'https://github.com/uyuni-project/terracumber.git'
+        terracumber_gitrepo = 'https://github.com/maximenoel8/terracumber.git'
+        terracumber_ref = 'support_tfvars'
+        terraform_init = true
+        rake_namespace = 'cucumber'
+        rake_parallel_namespace = 'parallel'
+        jenkins_workspace = '/home/jenkins/jenkins-build/workspace/'
+        pull_request_repo = 'https://github.com/uyuni-project/uyuni.git'
+        builder_api = 'https://api.opensuse.org'
+        build_url = 'https://build.opensuse.org'
+        builder_project = 'systemsmanagement:Uyuni:Master:PR'
+        source_project = 'systemsmanagement:Uyuni:Master'
+        sumaform_tools_project = 'systemsmanagement:sumaform:tools'
+        test_packages_project = 'systemsmanagement:Uyuni:Test-Packages:Pool'
+        build_repo = 'openSUSE_Leap_15.4'
+        environment_workspace = null
+        url_prefix="https://ci.suse.de/view/Manager/view/Uyuni/job/${env.JOB_NAME}"
```

Comment:
I'm not sure if I saw all the parameters to replace these values by SUMA values.